test/common_utils.py

```diff
@@ -311,7 +311,7 @@ def assertExportImportModule(self, m, args):
             results = m(*args)
         with freeze_rng_state():
             results_from_imported = m_import(*args)
-        self.assertEqual(results, results_from_imported, prec=3e-5)
+        self.assertEqual(results, results_from_imported, prec=3e-4)
```

Comment:
Without this change I keep getting failures on linux gpu for `maskrcnn_resnet50_fpn`

Reply:
Weird. The precision is still fairly small, so I'm ok with this change.